src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void toDrawFigure() {
```

Comment:
The method name 'toDrawFigure' does not accurately reflect the action it performs. According to the checklist, avoid using verbs for class/interface names, which also applies to method names that are conceptually part of an interface. A better name would be something like 'draw' to align with the checklist's guidance on naming.